common/djangoapps/student/views/dashboard.py

```diff
@@ -531,6 +533,21 @@ def _credit_statuses(user, course_enrollments):
     return statuses
 
 
+def show_load_all_courses_link(load_all_courses, user, courses_count):
```

Comment:
Nit: The common pattern of this module is to have the `user` as the first argument.